services/blockchain-indexer/shared/dataService/business/validator/index.js

```diff
@@ -14,7 +14,7 @@
  *
  */
 const { getValidator } = require('./validatorInfo');
-const { validateBLSKey } = require('./validateBLSKey');
+const { validateBLSKey } = require('./validatorValidateBLSKey');
```

Comment:
Please do not rename the file to `validatorValidateBLSKey` since it is already within the `validator` directory. The renaming was supposed to be only for the test files. Please revert this change.
```suggestion
const { validateBLSKey } = require('./validateBLSKey');
```

Reply:
applied.